data/spells/scripts/support/find_person.lua

```diff
@@ -46,6 +46,7 @@ function onCastSpell(creature, variant)
 	end
 
 	local playerPosition = creature:getPosition()
+	local targetPosition = targetPlayer:getPosition()
```

Comment:
@WibbenZ you don't get me, it will always be nill, doesnt matter if it has or not parameters